### DISCUSSION TITLE
Add shortcut to the "Add" buton present in the popup used to add a link

### DISCUSSION THREAD

**Discussion by hervehobbes** [Kovah/LinkAce#761] [Ideas & Feature Requests]
Hello,
I try not to use my mouse as much as I can, can you add a shortcut, for example Ctrl + A to the button used to add a link in the popup please?

**Comment by Kovah** [comment]
Could you specify which popup do you mean? The bookmarklet one?

**Comment by hervehobbes** [comment]
Yes, this one.
Thanks for your answer.
